Lesson 2/michael.pereverov/lesson_2.sh

```diff
@@ -0,0 +1,102 @@
+#!/bin/bash
+
+#1. Вывести список установленных программ и проверить, если в этом списке nginx.
+#  1.1. Если он есть. Удалить и вывести текст об удалении. Также указать, какая версия была удалена. 
+#  1.2. Если его нет, вывести текст что он не установлен в системе.
+RED='\033[1;31m'
+YLW='\033[1;33m'
+GRN='\033[0;32m'
+NC='\033[0m'
+
+APP_VERSION=1.14.2-1~bionic
+APP_NAME=nginx
+
+#Debian-like OS
+
+
+
+status=$(dpkg -s $APP_NAME 2>/dev/null | grep --color=auto -o 'installed')
+version=$(dpkg -s $APP_NAME 2>/dev/null | grep Version)
+if [[ "$status" == 'installed' ]]
+  then
+    echo -e "${RED}$APP_NAME${NC} is already Installed"
+    echo -e "$version\n"
+      sleep 1
+    echo -e "${YLW}Deletting...${NC}"
+      sleep 3
+    sudo apt purge -y $APP_NAME
+    echo -e "${GRN}Done!\n${NC}"
+      sleep 3
+  else
+    echo -e "${RED}$APP_NAME${NC} is Not installed\n"
+#    echo -e "${YLW}Installing...${NC}\n"
+#    sleep 3
+fi
+
+
+#2. Добавить внешнее репо nginx: (документация на репо. http://nginx.org/en/linux_packages.html#Ubuntu) и установить nginx 1.14.2.
+
+#echo -e "Update packages list:\n"
+#sleep 3
+#sudo apt-get update
+
+#echo -e "Install the prerequisites:\n"
+#sleep 3
+#sudo apt-get install curl gnupg2 ca-certificates lsb-release
+
```

Comment:
Adding repo keys...

E: gnupg, gnupg2 and gnupg1 do not seem to be installed, but one of them is required for this operation
(23) Failed writing body

Installing...